Player.py

```diff
@@ -42,6 +46,13 @@ def createImages(self):
     def draw(self, win):
         img = pygame.image.load(self.characterImg)
         win.blit(img, (self.x, self.y))
+        for bomb in self.bombs:
+            if bomb.timer >= 160:
+                bomb.__del__()
+                self.bombs.remove(bomb)
+                self.bombNumber += 1
+            else:
```

Comment:
So we're removing bomb and adding it? I don't get it

Reply:
I have already changed that